net/libreswan/files/etc/uci-defaults/901-libreswan

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
I don't think the creation of configurations is so good thing here!
I wouldn't expect that as a user. 
You have to think about on a firmware upgrade here.
Can we not solve this differently

Reply:
Done in https://github.com/openwrt/packages/pull/19233